src/FixedDataTableStore.js

```diff
@@ -11,10 +11,19 @@
 
 'use strict';
 
-import { createStore } from 'redux'
+import reducers from './reducers';
+import { configureStore } from '@reduxjs/toolkit';
+import { setAutoFreeze } from 'immer';
```

Comment:
Immer is not present in `package.json` but still it works because `@reduxjs/toolkit` has a dependecy on `immer`.

Reply:
Lets still include it since the standard practice is to list down all direct dependencies in package.json.

Reply:
Will they share the same immer instance. I am not sure

Reply:
If we use the same version of immer i.e. `immer@^8.0.1`, it will work fine. But, I am worried that if users use the same version, I guess it will affect their code as well